packages/server/src/controllers/questController.ts

```diff
@@ -53,42 +54,30 @@ export class QuestController extends BaseController {
         conversation: Conversation,
         quest: Quest,
         answer: Info
-    ): boolean {
-        const isQuestCompleted = quest.question.isAnswer(answer);
+    ): void{
         const conversationParticipants: Agent[] = conversation.participants;
         const questGiver = quest.giver;
         const questReceiver = quest.receiver;
         const room = conversation.room;
-
-        let questResultInfo;
+        const cc: ConversationController = new ConversationController();

```

Comment:
when you instantiate a nested controller, you need to pass in the main one so that the updates are all collected in the same map to be sent to client. I forgot to mention this, mb.
```suggestion
        const cc: ConversationController = new ConversationController(this);

```
Constructor for BaseController:
https://github.com/panoptyk/panoptyk-engine/blob/09e0e33958ce4be6e71b0b4cdbac3f193e1befba/packages/server/src/controllers/baseController.ts#L26-L32